tests/integration/seccomp.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load helpers
+
+SECCOMP_TESTS_BINDIR="$(mktemp -d "$BATS_TMPDIR/seccomp-bins.XXXXXX")"
+SECCOMP_TESTS=(
+	"seccomp_syscall_test1"
+)
+
+function setup() {
+	teardown_busybox
+	setup_busybox
+
+	# Compile the test cases...
+	for test_name in "${SECCOMP_TESTS[@]}"; do
+		gcc -static -o "$SECCOMP_TESTS_BINDIR/$test_name" "$TESTDATA/$test_name.c"
+	done
+}
+
+function teardown() {
+	teardown_busybox
+	rm -rf "$SECCOMP_TESTS_BINDIR"
+}
+
+@test "runc run [seccomp -ENOSYS handling]" {
+	update_config ".linux.seccomp = $(<"${TESTDATA}/seccomp_syscall_test1.json")"
+	update_config '.process.args = ["/seccomp_syscall_test1"]'
+
+	cp "$SECCOMP_TESTS_BINDIR/seccomp_syscall_test1" rootfs/
```

Comment:
So maybe just compile it right here with `gcc ... -o rootfs/test1` or so

Reply:
Will do.